pkg/actions/lua/storage/azure/azure.go

```diff
@@ -0,0 +1,18 @@
+package azure
+
+import (
+	"context"
+
+	"github.com/Shopify/go-lua"
+)
+
+func Open(l *lua.State, ctx context.Context) {
```

Comment:
please add docs for for the new azure lib, similar to [aws](https://docs.lakefs.io/howto/hooks/lua.html#lua-library-reference)

Reply:
Done

Reply:
Wonderful !